pep-0001.txt

```diff
@@ -278,24 +278,28 @@ for the discussion, so long as the the following criteria are met:
 * A direct link to the current discussion thread is provided in the PEP
   under the ``Discussions-To`` header.
 
-Typically, the `Python-Dev`_ mailing list and the
-`PEPs category`_ of the `Python Discourse`_ are good choices for most PEPs,
-while some specialized topics have specific venues, such as
+Typically, the `PEPs category`_ of the `Python Discourse`_
+is a good choice for most new PEPs,
+whereas historically the `Python-Dev`_ mailing list was commonly used.
```

Comment:
PEP 1 is very long, I'd suggest cutting the second part of this sentence for brevity
```suggestion
is a good choice for most new PEPs.
```

Reply:
I certainly agree that PEP 1 is rather long, and could do with being more shorter and focused at the macro level (and have some much more impactful ideas for this). That said, this is the micro level of less than a dozen words out of 6500; whether we cut out a relatively concise mention is more a question of whether it serves a purpose consummate with its fairly brief length.

In this case, I retained this line, the only reference left to Python-Dev in the PEP, since I felt it provided important context for historical PEPs, a large majority of which have their discussions and resolutions on that venue, many without one or both being explicitly mentioned/linked in the PEP in question.

However, if others feel that doesn't justify even the relatively brief mention, I can elide it as you suggest.